.gitignore

```diff
@@ -6,3 +6,10 @@ _static
 _templates
 
 TODO
+
+_trial_temp/
```

Comment:
This is all stuff created when running `tox` locally.

Reply:
The reason these aren't here already is because generally I'd recommend these go in your global git ignore, since they're things that should be ignored from all repos. E.g., mine is: https://github.com/Julian/dotfiles/blob/master/.config/git/ignore

But tbh I'm fine to merge it anyhow since I have had to explain ^ quite a few times so clearly at least some people are tripped up by it.